interface/forms/newpatient/common.php

```diff
@@ -422,8 +422,8 @@ function cancelClickedOld() {
      <td class='bold' nowrap><?php echo xlt('Onset/hosp. date:'); ?></td>
      <td class='text' nowrap><!-- default is blank so that while generating claim the date is blank. -->
       <input type='text' size='10' class='datepicker input-sm' name='form_onset_date' id='form_onset_date'
-       value='<?php echo $viewmode && $result['onset_date']!='0000-00-00 00:00:00' ? substr($result['onset_date'], 0, 10) : ''; ?>'
-       title='<?php echo xla('yyyy-mm-dd Date of onset or hospitalization'); ?>' />
+       value='<?php echo $viewmode && $result['onset_date']!='0000-00-00 00:00:00' ? oeFormatShortDate(substr($result['onset_date'], 0, 10)) : ''; ?>'
```

Comment:
The 0000-00-00 00:00:00 stuff here seems really odd. Does this every get outputted?